doc/yjit/yjit.md

```diff
@@ -165,9 +165,11 @@ YJIT supports all command-line options supported by upstream CRuby, but also add
 
 - `--yjit`: enable YJIT (disabled by default)
 - `--yjit-call-threshold=N`: number of calls after which YJIT begins to compile a function (default 30)
+- `--yjit-cold-threshold=N`: number of calls after which an ISEQ is considered cold and not
```

Comment:
We should add this to the help (`yjit_options` in ruby.c) as well. It would have more readers than this document.

Reply:
I guess we should implement that printing logic in options.rs and let ruby.c call it since we tend to forget it and it sometimes gets out of date.

Reply:
It would be neat if it could be automated somehow, the tricky bit is that you may want to have a shorter description string in the console output than you would in a readme file.